src/models/darp8/repairs.md

```diff
@@ -0,0 +1,211 @@
+- [Replacing an M.2/NVMe SSD](#replacing-an-m2nvme-ssd)
+- [Replacing the WiFi/Bluetooth module](#replacing-the-wireless-card)
+- [Replacing the CMOS battery](#replacing-the-cmos-battery)
+- [Replacing the fans/heatsink/thermal paste](#replacing-the-cooling-system)
+- [Replacing the keyboard](#replacing-the-keyboard)
+- [Replacing the battery](#replacing-the-battery)
+
+## Removing the bottom cover:
+
+Removing the cover is required to access the internal components. Prior to removing the cover, ensure the AC power is unplugged and all peripherals (including SD cards and USB drives) are unplugged or removed from the system.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 7 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to remove the bottom cover:
+
+1. Place the machine lid-side down.
+    - Use a soft surface (such as a towel) to avoid scratches.
+2. Remove the 16 bottom panel screws:
+    - 12 perimeter screws (thin), highlighted green below.
+    - 2 keyboard screws (thicker), highlighted yellow.
+    - 2 inner hinge screws (thickest), highlighted red.
+
+![Bottom panel screws](./img/bottom-screws.webp)
+
+3. Pull the bottom panel off, starting from the hinges in the back.
+
+## Replacing the RAM:
+
+The Darter Pro 8 supports up to 64GB (2x32GB) of DDR4 SO-DIMMs running at 3200MHz. If you've purchased new RAM, need to replace your RAM, or are reseating your RAM, follow these steps.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 10 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to replace the RAM:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Press the small tabs on both sides of the RAM simultaneously. The RAM should spring up to an angle.
+
+![RAM slots](./img/ram-slots.webp)
+
+3. Remove the RAM from the slot.
+4. Insert the new RAM (or reseat the existing RAM) by placing it into the keyed slot and pressing down until it clicks into place.
+
+## Replacing an M.2/NVMe SSD:
+
+This model supports two M.2 SSDs. Both slots are size 2280. SSD-1 supports PCIe NVMe Generation 3 or SATA III, while SSD-2 supports PCIe NVMe Generation 4.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 10 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>  
+
+### Steps to replace the M.2 drive:
+
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+2. Unscrew the retainer screw opposite the M.2 slot.
+    - SSD-1 is closest to the battery (pictured on the left.)
+    - SSD-2 is closest to the outer edge of the machine (on the right.)
+
+![M.2 slots](./img/m2-slots.webp)
+
+3. Remove the existing M.2 drive by pulling it out of the slot.
+4. Insert the new M.2 drive into the slot and hold it in place.
+5. Replace the retainer screw.
+6. If a square brace was present on the old drive, transfer it to the new drive.
```

Comment:
Does "square brace" refer to the thermal pad? I guess it's a pretty minor point, but it sounds a little odd to call it a brace.

Reply:
Yeah, I'll switch the wording to call it a thermal pad like we've started doing for other models. With some models it's been foam that seems more like a mechanical thing, other times it's the oily rubber more obviously for thermals.